src/talk/talk.window.js

```diff
@@ -44,11 +44,14 @@ function createTalkWindow() {
 
 	const window = new BrowserWindow({
 		...talkWindowOptions,
-		width: Math.min(1680, screenWidth),
-		height: Math.min(1050, screenHeight),
+		width: Math.min(1400, screenWidth),
+		height: Math.min(900, screenHeight),
 		show: false,
 	})
 
+	// Maximize window initially
+	window.maximize()
```

Comment:
Maybe only when screen size is not 4k?

Reply:
I can drop the commit (´。＿。｀)

Not sure, I want to make it depend on some the screen size. Even on 4k I personally always open it maximized

Reply:
yeah lets not do that then, until we store/restore the window size/position in the future :)

Reply:
Dropped